backend/internal/adapter/external/spotify_client_test.go

```
@@ -0,0 +1,92 @@
+func assertSpotifyDomainCode(t *testing.T, err error, want domain.ErrorCode) {
+	t.Helper()
+	var de *domain.DomainError
+	require.ErrorAs(t, err, &de)
+	assert.Equal(t, want, de.Code)
+}
+
+func TestSpotifyRequestToken(t *testing.T) {
+	t.Run("200 でトークンが返れば成功する", func(t *testing.T) {
+		srv := httptest.NewServer(http.HandlerFunc(func(w http.ResponseWriter, r *http.Request) {
+			w.Header().Set("Content-Type", "application/json")
+			w.Write([]byte(`{"access_token":"at","refresh_token":"rt","expires_in":3600}`))
```

golangci-lint (2.12.2)

[error] Error return value of `w.Write` is not checked

(errcheck)